src/App.vue

```diff
@@ -5,6 +5,13 @@
 		<b-container fluid id="app-body">
 			<router-view></router-view>
 		</b-container>
+		<footer class="footer">
+			<div class="container">
+				<span class="text-muted">
+					Fairdata services are offered by the Ministry of Education and Culture and produced by CSC - IT Center for Science
+				</span>
+			</div>
+		</footer>
 	</div>
 </template>
 
```

Comment:
How about making the footer stick to the bottom in short pages? E.g. something like this:

```
<style scoped>
	#app {
		display: flex;
		flex-direction: column;
		min-height: 100vh;
	}

	#app-body {
		flex-grow: 1;
	}

	.footer {
		padding: 16px;
		text-align: center;
	}
</style>
```

Reply:
good idea

Reply:
fixed